hypertrace-core-graphql-service/gradle.lockfile

```diff
@@ -118,7 +118,9 @@ org.hypertrace.core.serviceframework:platform-service-framework:0.1.70=compileCl
 org.hypertrace.core.serviceframework:service-framework-spi:0.1.70=compileClasspath,runtimeClasspath
 org.hypertrace.gateway.service:gateway-service-api:0.3.9=runtimeClasspath
 org.latencyutils:LatencyUtils:2.0.3=runtimeClasspath
+org.projectlombok:lombok:1.18.28=annotationProcessor
+org.projectlombok:lombok:1.18.30=compileClasspath
 org.reactivestreams:reactive-streams:1.0.3=compileClasspath
 org.reactivestreams:reactive-streams:1.0.4=runtimeClasspath
 org.slf4j:slf4j-api:2.0.7=compileClasspath,runtimeClasspath
-empty=annotationProcessor
+empty=
```

Comment:
This ok?

Reply:
Yep. Previously it had that to say there were no annotation processor dependencies, but that it was locked. Now the annotation processor includes lombok above.